src/pages/ResultDetail.tsx

```diff
@@ -51,17 +51,17 @@ function ResultDetail() {
 
   return (
     <GridContainer className="ResultDetail">
-      <div className="margin-y-2">
+      <div className="margin-y-2 display-flex flex-justify-between">
```

Comment:
i think replacing `flex-justify-between` w/ `flex-justify` should fix alignment

Reply:
yes yup ty ty